content/docs/deployment/csmoperator/_index.md

```diff
@@ -46,7 +28,13 @@ The installation process involves the creation of a `Subscription` object either
 
 ![OpenShit Operator Hub CSM install](operator_hub_install.gif)
 
->NOTE: Dell CSM Operator is distributed as both `Certified` & `Community` editions. Both editions have the same codebase and are supported by Dell Technologies, the only difference is that the `Certified` version is validated by RedHat. The `Certified` version is often released couple of days/weeks after the `Community` version.
+### Certified vs Community
+Dell CSM Operator is distributed as both `Certified` & `Community` editions. 
+
+Both editions have the same codebase and are supported by Dell Technologies, the only differences are:
+* The `Certified` version is validated by RedHat.
+* The `Certified` version is often released couple of days/weeks after the `Community` version.
+* The `Certified` version can only be installed on explicitely tested Openshift versions, while the `Community` version can be installed anywhere and with any Openshift version.
```

Comment:
Hi @coulof The third statement should be corrected to "Certified version is specific to Openshift and can only be installed on specific Openshift versions where it is certified. Community version is not specific to Openshift and can be installed on most of the k8s based container orchestrators "

Other differences that could be included:
- They are hosted on two different catalogs. Certified version on 'certified-operators' catalog and community version on 'community-operators' catalog
- Certified version is officially supported by Redhat by partnering with software vendors whereas community version is not officially supported by Redhat